src/libertem/viz/mpl.py

```diff
@@ -0,0 +1,78 @@
+import time
+import logging
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from .base import LivePlot
+
+
+logger = logging.getLogger(__name__)
+
+
+class MPLLivePlot(LivePlot):
```

Comment:
Would it be possible to only create the plot window when `display()` is called to keep it consistent with bqp?